sites/public/styles/headers.scss

```diff
@@ -292,14 +291,25 @@
   }
 }
 
-.hero__home-button {
+.hero__rentals-button {
+  color: black;
   background-color: #ffbe42;
 
   &:hover {
     @apply bg-primary;
     color: white;
   }
 }
+.hero__finder-button {
+  color: var(--bloom-color-white);
+  background-color: var(--bloom-color-primary-dark);
+  margin-left: var(--bloom-s2);
+  &:hover {
+    color: var(--bloom-color-white);
+    box-shadow: 0 0 0 var(--bloom-s0_5) var(--bloom-color-primary-dark) inset;
+    background-color: var(--bloom-color-black);
+  }
```

Comment:
The normal Detroit button on hover doesn't go to a darker color, but rather switches to an outlined version. Is this the standard that we want to keep? https://storybook.bloom.exygy.dev/?path=/story/actions-button--detroit-style

Reply:
@ludtkemorgan Talked about this one offline a bit with Jesse and he pointed me to these docs: https://www.figma.com/file/slSZvBKEUduz5ldBiDsIzE/detroit-style-guide?node-id=0%3A33. Seems like storybook dev will need to be updated though to reflect this.